jobsdb/jobsdb.go

```diff
@@ -1450,14 +1447,22 @@ func (jd *HandleT) constructParameterJSONQuery(table string, parameterFilters []
 	return fmt.Sprintf(`(%s.parameters @> '{%s}' %s)`, table, strings.Join(allKeyValues, ","), opQuery)
 }
 
+type cacheValue int
+
+const (
+	mayFindJobs cacheValue = iota
```

Comment:
Nitpicking - mayFindJobs name sounds a bit weird :) Call it hasJobs?

Reply:
Sure.